src/centos/scripts/start-marklogic.sh

```diff
@@ -220,7 +220,7 @@ elif [[ "${MARKLOGIC_INIT}" == "true" ]]; then
 
     # Make sure username and password variables are not empty
     if [[ -z "${ML_ADMIN_USERNAME}" ]] || [[ -z "${ML_ADMIN_PASSWORD}" ]]; then
```

Comment:
We should update these variables as well. And there are more instances of ML_ADMIN_... above.

Reply:
These variables are substituted by either env variables or secrets, hence we should keep this check as is. We can update the error message to "MarkLogic admin credentials must be set" instead of specifying the env var names in this error message.